[blog] Add post about remote (#41565)
mui/material-ui: docs/pages/blog/remote-award-win.md

---
title: MUI scoops the win in renowned Remote Excellence Awards
description: We're delighted to be honored with this global recognition for our commitment to fostering excellence in remote work.
date: 2024-03-20T12:00:00.000Z
authors: ['mikailaread']
tags: ['Company']
card: true
---

MUI has been named a **winner** in the first-ever [Remote Excellence Awards](https://remote.com/remote-excellence-awards/), in the Small & Mighty category! 🎉

<img alt="MUI's official winners badge provided by Remote." src="/static/blog/remote-award-win/award-image.png" width="2400" height="1100" />

## Small & Mighty winners

The awards, presented by [Remote](https://remote.com/), recognize companies that have demonstrated exceptional commitment, innovation, and enthusiasm in navigating the challenges and opportunities presented by remote work.

The awards celebrate MUI's efforts driving success in the realm of remote work and distributed teams and provide a roadmap of effective strategies and practices for other businesses to follow.

> The winners across 10 categories were selected through a rigorous evaluation process by a panel of judges, comprising Remote's leaders and a panel of influential industry figures, considering factors such as employee engagement, impact on employee engagement and productivity, and technology adoption.

Reviewer: Hero. You know I'm the queen of run on sentences. 🤣👑

> > MUI's achievement in the Remote Excellence Awards highlights the dedication and hard work of its leaders and teams in succeeding in the new world of work,

Author: oops

Said [Job van der Voort](https://www.linkedin.com/in/jobvo/), CEO, and Co-founder of Remote.

> Organizations like these have become the benchmark, through their commitment to excellence and their ability to thrive, paving a way forward for other organizations to confidently pursue their own remote work endeavors.

In judging, [Hailley Griffis](https://www.linkedin.com/in/hailleygriffis/), Head of Communications & Content at Buffer remarked:

> MUI is the definition of small and mighty. I am blown away by their commitment to building a company in their own way while achieving powerful results for themselves, their community, and their customers.

## Reflections from MUI's leadership

Being recognized as one of the best fully remote, distributed workplaces is, above all, a testament to the incredible dedication of our team.
A company's culture lives in the daily interactions and behaviors of the people who work there.

> I'm so proud to see ours celebrated for how — together — we've built a collaborative and inclusive remote work environment that's generating real value to a global community comprising hundreds-of-thousands of designers and developers.

Reviewer: Sooo much better.

<img alt="MUI's official winners badge provided by Remote." src="/static/blog/remote-award-win/award-image.png" width="2400" height="1100" />

## Small & Mighty winners

The awards, presented by [Remote](https://remote.com/), recognize companies that have demonstrated exceptional commitment, innovation, and enthusiasm in navigating the challenges and opportunities presented by remote work.

The awards celebrate MUI's efforts driving success in the realm of remote work and distributed teams and provide a roadmap of effective strategies and practices for other businesses to follow.
The winners across 10 categories were selected through a rigorous evaluation process by a panel of judges, comprising Remote's leaders and a panel of influential industry figures.
Factors considered included employee engagement, impact on employee engagement and productivity, and technology adoption.

> MUI's achievement in the Remote Excellence Awards highlights the dedication and hard work of its leaders and teams in succeeding in the new world of work

– [Job van der Voort](https://www.linkedin.com/in/jobvo/), CEO, and Co-founder of Remote.

> > Organizations like these have become the benchmark, through their commitment to excellence and their ability to thrive, paving a way forward for other organizations to confidently pursue their own remote work endeavors.

Author: good call, yeah

---
title: MUI scoops the win in renowned Remote Excellence Awards
description: We're delighted to be honored with this global recognition for our commitment to fostering excellence in remote work.

> date: 2024-03-21T00:00:00.000Z

Reviewer 2: Official annocement date

```suggestion
date: 2024-03-20T12:00:00.000Z
```

authors: ['mikailaread']
tags: ['Company']
card: true
---

MUI has been named a **winner** in the first-ever [Remote Excellence Awards](https://remote.com/remote-excellence-awards/), in the Small & Mighty category! 🎉

> <img alt="MUI's official winners badge provided by Remote." src="/static/blog/remote-award-win/award-image.png" width="1200" height="550" loading="lazy" />

Reviewer 2: Match the source. Also the image is in the viewport, so shouldn't be async.

```suggestion
<img alt="MUI's official winners badge provided by Remote." src="/static/blog/remote-award-win/award-image.png" width="2400" height="1100" />
```

Said [Job van der Voort](https://www.linkedin.com/in/jobvo/), CEO, and Co-founder of Remote.

> Organizations like these have become the benchmark, through their commitment to excellence and their ability to thrive, paving a way forward for other organizations to confidently pursue their own remote work endeavors.

In judging, Hailley Griffis, Head of Communications & Content at Buffer remarked:

> MUI is the definition of small and mighty. I am blown away by their commitment to building a company in their own way while achieving powerful results for themselves, their community, and their customers.

To learn more about the awards, visit [the Remote Excellence Awards page](https://remote.com/remote-excellence-awards/).
To learn more about us, visit [our About page](/about/).

## Honorary mention to our People & Culture leader

MUI is proud to also see our Director of People & Culture, [Mikaila Read](https://www.linkedin.com/in/mikailaread/), included for the second year running as a Top 50 Remote Accellerator in Remote's annual influencer report.

> <img alt="Mikaila wears a gray shirt with blue MUI logo to represent the team." src="/static/blog/remote-award-win/mikaila-remote-accelerator.png" width="1200" height="550" loading="lazy" />

Reviewer 2: ```suggestion
<img alt="Mikaila wears a gray shirt with blue MUI logo to represent the team." src="/static/blog/remote-award-win/mikaila-remote-accelerator.png" width="2400" height="1100" loading="lazy" />
```

Mikaila was recognized in the Remote Accelerator category which includes business, HR, and industry leaders actually making remote work happen at scale.

Regarding MUI's inclusion in the awards, Mikaila shared:

> Being recognized as one of the best fully-remote, distributed workplaces is, above all, a testament to the incredible dedication of our team.
> A company's culture lives in the daily interactions and behaviors of the people who work there.
> I'm so proud to see ours celebrated for how — together — we've built a collaborative and inclusive remote work environment that's generating real value to a global community comprising hundreds-of-thousands of designers and developers.
> It's a reflection of the excellence and commitment of every MUIer, and proof of the power of remote teams united by shared principles and values. Congratulations to all winners!

> When asked what he'd like to say to MUI employees about this win, Co-Founder and CEO Olivier Tassinari said:

Reviewer 2: To match the reference to Remote above

```suggestion
When asked what he'd like to say to MUI employees about this win, Co-founder and CEO Olivier Tassinari said:
```
